docker/tracker_dcs_stack/influxdb/config/influx-configs

```diff
@@ -0,0 +1,21 @@
+[default]
```

Comment:
I think that this file should be removed as it will be generated locally ?

Reply:
Was it really removed ? it doesn't seem so on github

Reply:
upps! sorry I think I push it before I delete it!

Reply:
done!